src/mainboard/pcengines/apu2/mainboard.c

```diff
@@ -45,6 +45,12 @@
 #endif //CONFIG_USE_CBMEM_FILE_OVERRIDE
 #include <cbfs_core.h>
 #include <spd_cache.h>
+#include <smbios.h>
+#include <string.h>
+
+#define NB_CFG_MSR (0xC001001f)
```

Comment:
@macpijan I don't like this hardcoded value. Quick search in code show that this value is already exposed by some header ie. [here](https://github.com/pcengines/coreboot/blob/apu2b-20160304/3rdparty/pi/amd/00730F01/AGESA.h#L1298). Please check registers values before adding them to code.